Cargo.toml

```diff
@@ -9,7 +9,10 @@ members = [
     "crates/pl011-driver",
     "crates/uart-driver",
     "crates/uart-interface-types",
```

Comment:
`uart-interface-types` doesn't need to be listed here either, right?

Reply:
Correct. Fixed.